### ISSUE TITLE
Some users cannot like, post, or comment

### ISSUE THREAD

**Issue by Veilstrom** [bluesky-social/social-app#7288]
### Steps to Reproduce

There isn't any way to reproduce this, as far as I can tell. Ever since the 1.96 update rolled out on the 19th, a number of people have been unable to like, comment, or reply with their accounts. This essentially makes their accounts unusable, since they can't interact with others in any way. Despite their posts not being publicly visible on Bluesky, they are visible on Clearsky. I only bring up the 1.96 update specifically because there has been no earlier date than the 19th that this has happened with.

### Attachments

For example, this user has their posts from the past few days visible on Clearsky, but their post from the 19th is the most recent one visible on their profile. Other people have had this happen to them on the 20th as well.
![Screenshot (191)](https://github.com/user-attachments/assets/bfb049e9-7684-444d-9140-f15f1ee2beee)
![Screenshot (192)](https://github.com/user-attachments/assets/7dfed698-06e1-401c-8494-5eeebf465de6)

The accounts affected by this are seemingly random, with no rhyme or reason as to why this would occur.

### What platform(s) does this occur on?

iOS, Android, Web (Desktop), Web (Mobile)

### What version of the app are you using?

Build version: 1.96.2; Bundle info: 3aa1a5f (prod); Bundle date: 24122022; Platform: web; Platform version: undefined

### Additional Information

Every instance of this bug that I've seen only occurred to users on the 19th and 20th.

**Comment by Veilstrom** [comment]
This issue is the same one mentioned in this report: https://github.com/bluesky-social/social-app/issues/7241#issue-2754458011

**Comment by oosterhagen** [comment]
Same as #2788

**Comment by oosterhagen** [comment]
7 days and counting...


Are we forced to return to X?

**Comment by Veilstrom** [comment]
Adding to this, but direct messages do work as intended. Everything else is still broken, as described.

**Comment by gaearon** [comment]
Just wanted to say the team is aware of some related issues and has been fixing some of them. If you’re experiencing it please keep adding details about which accounts are affected. The response times will be delayed until everyone’s back from vacation but it helps to have a checklist to verify the fixes against.

**Comment by wendyjf** [comment]
Reported and mentioned elsewhere but experiencing same on https://bsky.app/profile/wendyj.bsky.social
Just checked now, problem still ongoing. Likes, reposts, blocks, follow backs don't 'stick'
Since 19 December.

**Comment by davetroy** [comment]
Can confirm that I am having the same issue. Opened three tickets with support without response.
![IMG_0506](https://github.com/user-attachments/assets/50a64445-b310-47bd-a8e6-2761de7e45b9)

**Comment by devinivy** [comment]
One of our struggles digging into this is that we don't own any accounts affected by the issue, so it's a little tricky to perform certain kinds of debugging.  Could someone experiencing the issue run a quick experiment for me?

Use the "change handle" feature (Settings > Account > Handle) to update your handle, but just set it to its current value.  This should be a no-op for your account but it will push an identity event through the network.  I am interested to know if this has any effect on the issue for you— so please try to create a post, follow, or like afterwards and let us know if it goes through or remains stuck.

**Comment by wendyjf** [comment]
Updated my handle back to wendyj (so no change) 
Made a few Likes, and Reposts, and Replies. 
Tried to block, and follow, some folk 
None of these actions "stick" or remain ticked/selected on BSKY - 
BUT I can see two new blocks on Clearsky - even though on Bluesky these people are visible and have "follow back" buttons.

**Comment by wendyjf** [comment]
Happy to make my account, available to a programmer for debugging?  Its broken already so using it as a test-site is fine?

**Comment by JazRav** [comment]
> One of our struggles digging into this is that we don't own any accounts affected by the issue, so it's a little tricky to perform certain kinds of debugging. Could someone experiencing the issue run a quick experiment for me?
> 
> Use the "change handle" feature (Settings > Account > Handle) to update your handle, but just set it to its current value. This should be a no-op for your account but it will push an identity event through the network. I am interested to know if this has any effect on the issue for you— so please try to create a post, follow, or like afterwards and let us know if it goes through or remains stuck.

Have done this with https://bsky.app/profile/foxthing.gay swapped it to https://bsky.app/profile/old.foxthing.gay a week ago, softlocked out a week ago
![Screenshot_20241230-095926_Bluesky](https://github.com/user-attachments/assets/78a5ab30-16b3-40f2-9ae2-9cded2643afb)
![Screenshot_20220817-230555_Trebuchet](https://github.com/user-attachments/assets/1a678185-c171-4bcf-8d70-83ebb5f6f7e7)

**Comment by Lach-Arkwight** [comment]
I also have this problem. https://bsky.app/profile/lach.girlfailure.zone

**Comment by Panduhmonium-exe** [comment]
My account has the same problem. https://bsky.app/profile/panduhmonium.bsky.social

I cannot post, comment, reply, like, unlike, follow, unfollow, block, or unblock. If I look at ClearSky like some people have mentioned I can see the test post I made on December 24th and 25th. But those two post do not show up on my Bluesky...only on ClearSky. Last post that shows on my account on Bluesky is from either December 19th or 20th.

**Comment by wendyjf** [comment]
As with the above, my last posts showing up on my BSky account, were December 19th 
maybe a code change that happened on the 19th was responsible for this bug?

**Comment by jobrodie** [comment]
My friend's account (@alomshaha.bsky.social) is similarly affected. He can't post or like but can see DMs and other people's posts. His 'failed' poss are also showing up on Clearsky too https://clearsky.app/alomshaha.bsky.social/history. I've made him aware of this Github post so he knows it's a problem affecting others too. Fingers crossed for a fix, thanks, Jo.

**Comment by oosterhagen** [comment]
Tried the solution as provided by Devinivy

>... update your handle, but just set it to its current value. This should be a no-op for your account but it will push an identity event through the network.

in my case set it from @oosterhagen.bsky.social to @oosterhagen.bsky.social but it remained stuck.
then set it from @oosterhagen.bsky.social to @0osterhagen.bsky.social with the same result.
Quite astounded that one can set his own handle without any check if it exists. Could it be that there are more users with the same handle? 

Also very surprised that issue https://github.com/bluesky-social/social-app/issues/6528 was closed!

Without any resolution ...

**Comment by oosterhagen** [comment]
Spurious other mentions of the bug found. BSky has a problem ...

https://github.com/bluesky-social/social-app/issues/7241
https://github.com/bluesky-social/social-app/issues/6528
https://github.com/bluesky-social/social-app/issues/2788

**Comment by brianolson** [comment]
If anyone wants to make some traffic _right now_ and for the next hour or two I'm watching the logs US/Eastern 2-4ish. Let me know your handle and about the time you tried to make a post. Thanks.

**Comment by davetroy** [comment]
I just did a couple of likes, and a post on account @davetroy.com / did:plc:hhaay6m47grq5cyim2fjphpb

Been having this problem since Dec 20.

**Comment by kirahowe** [comment]
> If anyone wants to make some traffic _right now_ and for the next hour or two I'm watching the logs US/Eastern 2-4ish. Let me know your handle and about the time you tried to make a post. Thanks.

I just tried around 2:39PM ET

**Comment by Panduhmonium-exe** [comment]
Posted at https://bsky.app/profile/panduhmonium.bsky.social right now at 2:53 pm EST

**Comment by wendyjf** [comment]
> If anyone wants to make some traffic _right now_ and for the next hour or two I'm watching the logs US/Eastern 2-4ish. Let me know your handle and about the time you tried to make a post. Thanks.

Just posted "this is a test" on https://bsky.app/profile/wendyj.bsky.social

**Comment by brianolson** [comment]
One more try?

**Comment by wendyjf** [comment]
Just sent another test post https://bsky.app/profile/wendyj.bsky.social

**Comment by davetroy** [comment]
Just did another.

**Comment by Veilstrom** [comment]
> If anyone wants to make some traffic _right now_ and for the next hour or two I'm watching the logs US/Eastern 2-4ish. Let me know your handle and about the time you tried to make a post. Thanks.

My friend https://bsky.app/profile/aucatgirl.bsky.social tried to make a test post, but it won't even show up in the media tab on clearsky like it used to when they were first effected by this bug

**Comment by Veilstrom** [comment]
If it helps, one of my friend's posts did go through when the bug occurred, but it came with this notice. The post also had a picture that wouldn't appear on mobile, but would appear on desktop. They've deleted the post since there wasn't any point in keeping it up, but I still have a screenshot of the notice
![Screenshot_20241221_163520_Chrome](https://github.com/user-attachments/assets/09823fe4-1376-4733-9252-e29a86614e2a)

**Comment by AndreLevin** [comment]
I also have one account with this bug. Two reposts and one post went through, but nothing since then.
![image](https://github.com/user-attachments/assets/91cec637-e67b-404e-99b0-5afc2ab8f513)

The account has been broken for close to three weeks now

**Comment by kirahowe** [comment]
I'm sure people are looking into this and I understand it affects only a small number of accounts so may not be a huge priority, but I just wanted to check whether there are any updates or recommended steps for those of us impacted? Accounts that are affected are effectively locked out.

Are there any indications this might be resolvable, or any troubleshooting steps we can take?

**Comment by oosterhagen** [comment]
I'm not so sure people are looking into this.

And it doen't mtter if only a mall number of account are affected. If you're affected it means a lot. Like not having a proper SM at your fingertips ...

**Comment by davetroy** [comment]
@brianolson is there a status update on this? Anything we can do to help troubleshoot?

**Comment by oosterhagen** [comment]
OK, tried another account, can't because mail account is already registered. Cleaned my account, all feads gone. No avail.

Tried to remove a "list". Impossible because: "Could not locate record: at://did:plc:kzacdegkbs2yymtyjghmjx3t/app.bsky.graph.list/3lbezcj6vfi2e"

**Comment by oosterhagen** [comment]
AAARGH! Deleted my account to revive. I can't because the handle is taken!

so far there were 8 nice months but BlueSky is getting nasty!

**Comment by jobrodie** [comment]
Another person whose Bluesky account has been 'locked' for the last three weeks is https://bsky.app/profile/egobot.bsky.social - they got in touch after seeing my [blog post about this](https://howtodotechystuff.wordpress.com/2025/01/01/lots-of-people-cant-post-on-bluesky-its-a-known-glitch/) and asked me to add their handle here for the collection. If anyone else is reading and wants their a/c added (and doesn't have a Github a/c) I'm on jo.brodie@gmail.com or @jobrodie on Twitter.

Personally I'm unaffected by the glitchy update but I'd be feeling a bit glum if I couldn't use Bluesky for this length of time and no updates or indications on what the problem is or how long it might take to be resolved. 

Jo

**Comment by jobrodie** [comment]
> AAARGH! Deleted my account to revive. I can't because the handle is taken!
> 
> so far there were 8 nice months but BlueSky is getting nasty!

I wonder if the update that caused the 'can't post' glitch might be the one that's also prevented you from deleting and restarting your account? This post from Mashable (20 Dec) highlights that one of the recent update's planned effects was to fix cybersquatting that arose from people starting with a regular Bluesky account then swapping to a domain-verified one. They'd then lose their old 'bsky.app' handle which someone else could pinch. Now the two are linked so no impersonation on a previous handle.

Bluesky cybersquatting problem addressed in latest update
https://mashable.com/article/bluesky-cybersquatting-problem-addressed-latest-update

Jo

**Comment by davetroy** [comment]
If it is related to a cybersquatting problem, I did in fact sign up at davetroy.bsky.social and then verified my own domain handle as @davetroy.com; I was chagrined to learn that someone else had taken davetroy.bsky.social later. This user has that name so has a claim to it, but it's annoying given that I did claim it first and it has led to some confusion from users looking to find me and finding him instead. Perhaps this could be the cause of some of the trouble?

**Comment by jobrodie** [comment]
Possibly, though the person who got in touch today (whose handle I've added in an earlier post) is on a regular Bsky account so ... possibly not. It's a bit of a puzzle!

Jo

**Comment by amayii0** [comment]
Hi all,
Experiencing same issue since around 20 dec.
My handle is https://bsky.app/profile/amayii.bsky.social
I've been unable to change my handle for some workaround explained higher
Good luck to the team

**Comment by kirahowe** [comment]
> If it is related to a cybersquatting problem, I did in fact sign up at davetroy.bsky.social and then verified my own domain handle as @davetroy.com; I was chagrined to learn that someone else had taken davetroy.bsky.social later. This user has that name so has a claim to it, but it's annoying given that I did claim it first and it has led to some confusion from users looking to find me and finding him instead. Perhaps this could be the cause of some of the trouble?

That's an interesting thread to pull on.. I also originally signed up as @kirahowe.bsky.social and then subsequently verified (and then used) @kirahowe.com.. is there any pattern to the affected accounts? Like maybe verifying a domain that was the same as the original handle or something?

> I'd be feeling a bit glum if I couldn't use Bluesky for this length of time and no updates or indications on what the problem is or how long it might take to be resolved

This captures my sentiment well.. I'm not upset or anything and I completely sympathize with the developers who must be completely overwhelmed by the sudden explosion of users, but this is the type of bug that you'd want to get to the bottom of early on before the platform grows much more since it's only a matter of time before someone influential enough to affect the platform's reputation is affected. I just feel kind of bummed because I was really excited about Bluesky and it was going really well, plus I kind of rallied a bunch of people in my weird little niche community to join -- not that that's the sole reason they did, but it's just I was pretty bullish about Bluesky as it was exploding and now the joke's on me for being so optimistic :)

Anyway I would be posting and using Bluesky most days if I could, I still hope this gets resolved so I can go back.

**Comment by Veilstrom** [comment]
Changing domains being involved is an interesting idea, but I know for a fact that my friend has kept the default bsky.social domain since their account was created. Not to say that it isn't related, but it doesn't seem like the common thread connecting these accounts to this bug.

I also share the same sentiment as others here. While I'm glad the bluesky team is aware of the bug and are working to fix it, I am upset by the lack of formal responses to this issue (both in terms of report tickets and the main bsky account). I imagine most people wouldn't think to check the github or the devs personal accounts for information, which makes it look like they aren't paying attention - even if they are. It just feels like something that could dampen the goodwill of the platform, is all. Best of luck to the devs

**Comment by wendyjf** [comment]
Yeah also disappointed with the lack of feedback on this from Blue SKy

I reported the issue formally, through the Blue Sky help channel, around Christmas - before finding you all here on Github, 
no formal response yet, at all. 

inactive now for 2 weeks - a long time on socials.

And I haven't verified a domain or changed handles, I'm using the same handle as registered in 2023. https://bsky.app/profile/wendyj.bsky.social

**Comment by JazRav** [comment]
https://bsky.app/profile/old.foxthing.gay/post/3ldrgzxxag227

One of the post i made since this big began started to show in my following feed but no one can see it nor can i see it on my profile 
![Screenshot_20250106-143403_Bluesky](https://github.com/user-attachments/assets/20fada76-75d6-4aa5-9c3b-28257401c8e7)

**Comment by altryism** [comment]
I've been experiencing the same thing since 12/19/24. unable to post/like/etc. account:

https://bsky.app/profile/altryism.bsky.social

**Comment by tomhaley** [comment]
Same problem since 12/20, also did the handle change to verified domain a little while ago

https://bsky.app/profile/thomashaley.net

**Comment by brianolson** [comment]
We believe we have found the problem, about a million posts and likes sitting in a broken process, and the backlog should clear in a few hours

**Comment by Veilstrom** [comment]
> We believe we have found the problem, about a million posts and likes sitting in a broken process, and the backlog should clear in a few hours

I can confirm, one of my friend's posts from the 20th just showed up on their page. Their reply to my comment on that post hasn't shown up on bluesky yet, but it is visible on clearsky, so hopefully it should appear properly in the next few hours

**Comment by Lach-Arkwight** [comment]
Sounds like we'll be able to use our accounts again soon then! If we still
don't, I guess we make another post here?

**Comment by ericvolp12** [comment]
Current burndown rate should have things caught up in the next 2-4 hours.
Sorry for the issues y'all experienced, we were a bit short-staffed over the holidays but are back in action now!

**Comment by jobrodie** [comment]
That's very encouraging Bluesky team - thank you for updating :-)
Jo

**Comment by Veilstrom** [comment]
One more update, everything seems to be in order now! My friend has finally been able to like, post, and comment again. I've also seen the backlog of posts appear on other people's accounts too.

**Comment by amayii0** [comment]
Update : fixed for me!

**Comment by jobrodie** [comment]
> Current burndown rate should have things caught up in the next 2-4 hours. Sorry for the issues y'all experienced, we were a bit short-staffed over the holidays but are back in action now!

@ericvolp12 My reaction was meant to be a big beaming grin emoji and not a laughing one!

Jo :-D

**Comment by fluffy-critter** [comment]
These two bridged profiles of mine are still stuck with their last updates being over a month ago:

https://bsky.app/profile/beesbuzz.biz
https://bsky.app/profile/sockpuppet.band

They're both IndieWeb sites bridged via brid.gy. The dashboard shows those sites as updating correctly (and they've been appearing on their respective Fediverse feeds correctly as well):

https://fed.brid.gy/web/beesbuzz.biz
https://fed.brid.gy/web/sockpuppet.band

**Comment by oosterhagen** [comment]
to late for me. I deleted my account. Started a new one. chenged the handle to oosterhagen.bsky.social and went head.
Lost lots of posts but at least I'm back again.

And there's a nice little yellow sun on my account...

**Comment by kerim** [comment]
I have a bridged RSS feed that is stuck. Because it is an RSS feed I can't really change the handle. 

Details here:

https://github.com/snarfed/bridgy-fed/issues/1713#issuecomment-2608636796

**Comment by zacuke** [comment]
I seem to be running into this issue:  I can't edit my profile and posts don't show under my profile.

I see the posts when I call `/xrpc/com.atproto.repo.listRecords?collection=app.bsky.feed.post` directly.

But not when I call `/xrpc/app.bsky.feed.getAuthorFeed`  which returns empty `feed` array.  I pulled this call from the browser debugger. So, the website seems to be showing empty posts under profile because `getAuthorFeed` is returning empty.

I'm using self hosted PDS. Things are installed completely vanilla per the official setup instructions and script. I've tried reinstalling and recreating everything twice, but profile and posting seem to do nothing with no discernible errors. I've also tried creating additional users on the PDS but every user seems to have this same issue.

Should I open a new issue on the atproto repo?

**Comment by estrattonbailey** [comment]
Since it seems the original issue reported was solved, I'm going to close this. Feel free to add comments if you feel it's still a problem. If you're having a different (or only potentially related) issue, please create a new issue at the top of this page.

**Comment by AntoineTwan** [comment]
Not completely sure it's exactly the same issue as it was supposed fixed but as my new posts are displayed in Clearsky but not Bluesky I guess it's something close. 
https://github.com/bluesky-social/social-app/issues/9071

**Comment by luxurystorecc1** [comment]
Reno Performance: Merging Desert Durability with Modern Athletic Fashion
Reno is a city that bridges the gap between urban development and raw mountain adventure. Gear here must be versatile enough to transition from the downtown courts to the trails of the Sierra Nevada. For basketball enthusiasts in the 775, [iconic silhouette basketball sneakers for street style](https://www.google.com/search?q=https://www.jordan1s.com/) provide the necessary durability and comfort for both indoor and outdoor performance.

Football is a major pillar of the community in Northern Nevada. Fans often look for [discounted authentic-style pro jerseys](https://www.cheapnfl-jerseys.com/) to represent their teams during intense game days. To combat the desert sun, [professional basketball team hats and caps](https://www.google.com/search?q=https://www.cheapnfl-jerseys.com/collections/headwear-nba-headwear) are essential gear for any fan spending time outdoors.

Luxury Leisure and Heat Management
The upscale communities in Reno prioritize performance and aesthetics. Golfers frequenting local clubs often choose [premium moisture-wicking casual apparel](https://www.peter-milla.com/) for its ability to stay cool and sharp. For casual days exploring the city, [innovative lightweight knit lifestyle shoes](https://www.yeezyboost-350.com/) are a top footwear choice for residents who value comfort.

Soccer’s popularity is immense in Reno, with fans opting for [official major league soccer team kits](https://www.google.com/search?q=https://www.soccerjerseysstore.com/collections/mls-jerseys). Competitive swimming is a year-round activity at local aquatic centers, requiring [professional racing swimsuits and goggles](https://www.swim-outlet.com/) for peak performance. For formal events, [elegant white formal and cocktail dresses](https://www.google.com/search?q=https://www.cheapdressesformal.com/collections/dresses-white) are a seasonal favorite for those attending galas or weddings.

Outdoor enthusiasts often reach for [unisex vintage style team caps](https://www.google.com/search?q=https://www.cheapnfl-jerseys.com/collections/baseball-caps-baseball-caps-for-men-and-women) for a relaxed weekend look. To show support for the winter sports season, [official league-certified hockey merchandise](https://www.custombaseball-jerseys.com/) is a high-demand item. For gala attendees, a [vibrant crimson dress for evening guests](https://www.swim-outlet.com/collections/dresses-red) offers a striking look for any high-end occasion.
